flocker/cli/functional/test_sshconfig.py

```diff
@@ -64,7 +64,13 @@ def test_connection_failed(self):
 
         exc = self.assertRaises(CalledProcessError,
                                 self.configure_ssh, b"127.0.0.1", port)
-        self.assertIn(b"refused", exc.output)
+        # There are different error messages on different platforms.
```

Comment:
A quick search shows nothing and I don't know - I'm not going to dig into this although I agree that it would be nice.